pkg/processor/procbuiltin/unwrap_test.go

```diff
@@ -42,7 +42,7 @@ const DebeziumRecordPayload = `{
 		 "schema": {} 
 		}`
 
-const OpenCDCRecordCreatePayload = `{
+const OpenCDCRecordWithAfterPayload = `{
```

Comment:
I like the idea behind this PR! What I'd suggest changing are the names to include the operation/test case as well. E.g. create and update always have an after, update sometimes has a before, delete has neither.

Reply:
@hariso updated, let me know what you think https://github.com/ConduitIO/conduit/pull/1363/commits/84e58c05e060fb405010c45a8972805c706a83e1.